default.nix

```
@@ -34,13 +40,17 @@ rec {
   }).config.modules-docs;
 
   # Docs
-  docs = pkgs.callPackage ./docs { inherit modules-docs; };
+  docs = nixpkgs.callPackage ./docs { inherit modules-docs; };
 
   # Tests
-  tests = import ./tests { inherit pkgs system; };
+  tests = import ./tests {
+    inherit system;
+    inputs = null;
+    pkgs = nixpkgs;
```

```suggestion
    inherit nixpkgs system;
    inputs = null;
```
pass nixpkgs the same around?

---

Ok for tests, but once crossing the module system, maybe stick with the current nomenclature 

If we want to be fancy, we can put `pkgs` behind a warning. But maybe just not yet.

---

Ok, let me stop be pedantic :)